src/lib/model/lib/shared/index.js

```diff
@@ -231,7 +233,7 @@ const internalTransactionRequestResponseToMojaloop = (internal) => {
  */
 const mojaloopPrepareToInternalTransfer = (external, quote) => {
     let internal = null;
-    if(quote) {
+    if (quote) {
```

Comment:
These unnecessary whitespace changes make diffs incredibly large and hard to read.